sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Customizations/Models/AssetConversionProperties.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    /// <summary> Represents the status of an AOA asset conversion job. </summary>
+    [CodeGenModel("IngestionProperties")]
+    public partial class AssetConversionProperties
+    {
+        internal AssetConversionProperties()
+        {
+        }
+
+        /// <summary> The URI for downloading the generated AOA Model. </summary>
+        public Uri OutputModelUri
+        {
+            get
+            {
+                return new Uri(OutputModelUriString);
+            }
+        }
+        /// <summary> The Uri to the Asset to be ingested by the AOA Asset Conversion Service. This asset needs to have been uploaded to the service using an endpoint provided from a call to the GetUploadUri API. </summary>
+        public Uri InputAssetUri
+        {
+            get
+            {
+                return new Uri(InputAssetUriString);
+            }
+            internal set
+            {
+                InputAssetUriString = value.AbsoluteUri;
+            }
+        }
+
+        /// <summary> The status of the AOA asset conversion job </summary>
+        [CodeGenMember("JobStatus")]
+        public AssetConversionStatus? ConversionStatus { get; internal set; }
```

Comment:
Formatting: Please add lines separating the properties. It make it easier to look at.

Reply:
Pending

Reply:
Lines added between properties